wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ConversationListViewController+NavigationBar/ConversationListViewController+NavigationBar.swift

```diff
@@ -213,9 +214,34 @@ extension ConversationListViewController {
         }
 
         action.setValue(attributedTitle, forKey: "attributedTitle")
+        action.accessibilityLabel = setAccessibilityLabelForFilterAction(for: filter, isSelected: isSelected)
+
         return action
     }
 
+    func setAccessibilityLabelForFilterAction(
+        for filter: ConversationFilterType?,
+        isSelected: Bool
+    ) -> String {
```

Comment:
```suggestion
    func accessibilityLabelForFilterAction(
        for filter: ConversationFilterType?,
        isSelected: Bool
    ) -> String {
```

Reply:
because it returns the value, it's better to remove the set

Reply:
Done: https://github.com/wireapp/wire-ios/commit/977d42af1e5e41d40c6d289f4d80a0ab8f4c6a83